pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -243,15 +243,18 @@ rustPlatform.buildRustPackage rec {
     runHook postInstall
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "zeditor";
```

Comment:
I think this needs the full path since the script will not have `$out/bin` in its `PATH`

Reply:
Indeed, it's all good now :